apps/artwork/components/bid/query/me.coffee

```diff
@@ -1,8 +0,0 @@
-module.exports = """
```

Comment:
Seems like we don't need this anymore given that we never reference `me.bidder_positions`